### PR TITLE
remove method which breaks debuggin in python 3

### PR DESCRIPTION
**Pull request by alexpantyukhin**
#1037

### CODE REVIEW

File: werkzeug/debug/console.py [hunk 1]
```diff
@@ -175,12 +175,6 @@ def runsource(self, source):
             output = ThreadedStream.fetch()
         return prompt + escape(source) + output
 
-    def runcode(self, code):
```
Comment (untitaker) [inline werkzeug/debug/console.py]: I think it's important to understand what this method was overriding in Python 2. I suspect you'll now get nonsense behavior if you trigger a traceback in the Werkzeug debugger console.

**Comment by alexwlchan** [comment]
Removing this method fixes my specific problem, but introduces a new one – tracebacks are now incomplete. Here’s what happens if I use the release version of Werkzeug, deliberately triggering two exceptions in the interactive debugger:

screen shot 2016-11-12 at 20 18 58

Here’s the same code if I apply your patch:

screen shot 2016-11-12 at 20 18 06

File: werkzeug/debug/console.py (hunk 1 above)
Comment (alexwlchan) [inline werkzeug/debug/console.py]: I confess I don’t fully understand what this method was overriding, and the [original commit](https://github.com/alexpantyukhin/werkzeug/commit/4ba78986452c322c95fe600409814e4c3db85040) is over nine years old.

I notice that the standard library version of this method uses `exec`, not `eval` (see [code.py L91](https://hg.python.org/cpython/file/tip/Lib/code.py#l91)). Swapping this out for `exec(code, self.locals)` does fix my initial problem, but introduces the same problem of empty tracebacks as this patch.

Comment (alexpantyukhin) [inline werkzeug/debug/console.py]: Yeah..  I see. interesting difference between versions of python.

**Pull request closed without merging**